### ISSUE TITLE
RHD RTL support

### ISSUE THREAD

**Issue by miguel-galarza** [google/automotive-design-compose#1350]
Currently there is no automatic support to RHD (Right-Hand Drive) and RTL (Right-to-Left) in designCompose, and therefore multiple Figma files should be created for rendering different layouts. Is this feature planned?

**Comment by iamralpht** [comment]
That's correct -- we'd recommend using variants at the root layer to achieve RHD and RTL, because automatic mirroring doesn't get the right result in cars. Would this work for you @miguel-galarza ?

We could specify the keywords/enums for RHD and RTL as part of the DesignCompose framework, rather than the application needing to specify it itself.

**Comment by miguel-galarza** [comment]
Hi, thank you for the reply, yes this could a way to implement it.

**Comment by anshu10sep** [comment]
Right now there is no plan on adding explicit support for RHD into DC
